### DISCUSSION TITLE
Respond direct messages with quick reply options

### DISCUSSION THREAD

**Discussion by jestsee** [tweepy/tweepy#1742] [Questions] [answered]
Hi there,

I am trying to respond direct message from someone who provides quick reply options (using api.send_direct_message function). I want to reply their direct messages with options that they provided instead reply it with my own text. If I'm not mistaken, responding through quick reply options will pass quick_reply_response object but I can not find in documentation how I can pass this object. I want to validate if I am missing something or there is no support yet to use this by tweepy.

**Comment by Harmon758** [comment, marked as answer]
If I'm understanding correctly, you're asking how to select a quick reply option in a direct message from someone.
Unfortunately, Twitter's API doesn't not support doing this.
You can provide feedback requesting this on the [Twitter Developers Forums](https://twittercommunity.com/) or through the [Twitter Developer Feedback site](https://twitterdevfeedback.uservoice.com/forums/930250-twitter-api).
